main.tf

```diff
@@ -1,25 +1,45 @@
 terraform {
   required_providers {
     azurerm = {
-      source = "hashicorp/azurerm"
+      source  = "hashicorp/azurerm"
       version = "3.105.0"
     }
   }
 }
```

Comment:
This code should be better to move in `provider.tf` too

Reply:
Done.